specification/cognitiveservices/data-plane/EntitySearch/v1.0/EntitySearch.json

```diff
@@ -0,0 +1,1018 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Bing Entity Search API",
+    "description": "The Entity Search API lets you send a search query to Bing and get back search results that include entities and places. Place results include restaurants, hotel, or other local businesses. For places, the query can specify the name of the local business or it can ask for a list (for example, restaurants near me). Entity results include persons, places, or things. Place in this context is tourist attractions, states, countries, etc.",
+    "version": "1.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{azureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "$ref": "#/parameters/AzureRegion"
+      }
+    ]
+  },
+  "parameters": {
+    "azureRegion": {
```

Comment:
...this guy to `AzureRegion` (while leaving the name below as is)

Reply:
d'oh.

Reply:
Thanks for the multiple reviews and quick turnaround mate, really appreciate it :)